onnxruntime/core/framework/session_state.cc

```diff
@@ -776,6 +778,26 @@ Status SessionState::LoadFromOrtFormat(const fbs::SessionState& fbs_session_stat
 }
 #endif
 
+// Calculate the use count of a constant initialized tensor, including the use in subgrah.
+// Note: This function doesn't handle this case below:
+// The main graph has a constant initializer called X, and the subgraph also has a constant initializer called X, which overrides the X from main graph.
+// For case like this, the current implementation will calculate an use count of 2, but they could contain completely different values so each should have a use count of 1.
+// This is a very rare case. If it happends and X is prepacked, the consequence is that X won't be released and memory usage of X won't be saved. This will be fine.
```

Comment:
nit: subgrah -> subgraph, happends -> happens

[https://marketplace.visualstudio.com/items?itemName=EWoodruff.VisualStudioSpellCheckerVS2017andLater](https://marketplace.visualstudio.com/items?itemName=EWoodruff.VisualStudioSpellCheckerVS2017andLater) is very helpful